src/python/tests/database/test_dbconnection_lite.py

```diff
@@ -0,0 +1,116 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit testing of `ensembl.io.genomio.database.dbconnection_lite` module.
+"""
+
+from pathlib import Path
+from typing import Optional
+from unittest.mock import patch
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.engine import URL
+from sqlalchemy.orm import Session
+
+from ensembl.io.genomio.database import DBConnectionLite
+from ensembl.core.models import Base, Meta
+
+_METADATA_CONTENT = {
+    "species.scientific_name": ["Lorem Ipsum"],
+    "species.taxonomy_id": ["123456"],
+    "species.classification": ["Insecta", "Lorem"],
+}
+
+
+# Create a SQLite database fixture with only a meta table and limited data
+@pytest.fixture(name="db_file", scope="session")
+def db_file_test(tmp_dir: Path) -> Path:
+    """Get a path to a db file."""
+    test_db_file = tmp_dir / "tmp_sqlite_core.db"
+    return test_db_file
+
+
+@pytest.fixture(name="db_engine", scope="session")
+def db_engine_test(db_file: Path):
+    """Get a SQLalchemy engine to a populated database."""
+    db_url = f"sqlite:///{db_file}"
```

Comment:
Just for future reference, in the ensembl-py plugin there is already a global variable with the selected server you may want to use for your tests. I know it is not applicable in this case, but worth flagging that up.